File: week6/done/mario_easy/mario_easy.py
from cs50 import *


while True:
    number = get_int("Height: ")
    if number <= 8 and number >= 1:
        break

for i in range(number):
    print(" " * (number-(i+1)), end="")
    print("#" * (i+1), end="")
    print()

File: week1/done/mario/mario_easy.c
// gcc -o mario_easy mario/mario_easy.c cs50.c
// ./mario_easy

#include <stdio.h>
#include "cs50.h"

int main(void)
{
    int a = 1;
    int num = get_int("Height: ");
    while (num > 8 || num < 1){
        while (num > 8 || num < 1)
        {
            num = get_int("Height: ");  
        }  
    printf("Stored: %d\n",num);    
    } 

    while (a <= num)
    {
        for (int b = 0;num-b > a;b++)
        {
            printf(" ");
        }
        for (int b = 0;b < a;b++)
        {
            printf("#");
        }
    printf("\n");   
    a++;
    }
}